python/utils/worker.py

```diff
@@ -163,8 +169,10 @@ def producer(
             assert chain_id is not None, "[Parser] Chain Id doesn't exist"
             q.put((chain_id, response.transactions))
 
+            # This is not accurate because queue in python is unbounded thus
+            # no time is spent waiting for the queue to be available.
```

Comment:
The below log is more related to waiting for `response = next(response_stream)` to finish than putting the txn's in the queue.

Reply:
yea, updated the log.